lib/hyrax.rb

```diff
@@ -40,6 +40,18 @@ module Hyrax
     autoload :Listeners
   end
 
+  ##
+  # @return [GlobalID]
+  # @see https://github.com/rails/globalid
+  def self.GlobalID(input) # rubocop:disable Naming/MethodName
+    case input
+    when Valkyrie::Resource
+      ValkyrieGlobalIdProxy.new(resource: input).to_global_id
+    else
+      input.to_global_id if input.respond_to?(:to_global_id)
```

Comment:
Do we need an exception raised? We'll be returning `nil` if the input doesn't respond to `:to_global_id`.

Reply:
i actually tried to remove this, and it was a quagmire involving `PowerConverter` backward compatibility. 

what i'd really like to do in the `else` case is return a `NullGlobalID` but i think that would involve touching pretty much every part of `Hyrax::Workflow` and `Sipity`.

Reply:
Sounds like we ship it!